src/main/java/florizz/core/FlowerDictionary.java

```diff
@@ -56,8 +56,8 @@ public static void startup() {
 
         // [Fillers have yet to be implemented]
         add("Baby Breath", "White", new String[]{"Wedding", "Valentines", "Mothers Day"}, 1.00,
-                new String[]{"Innocence", "Kindness", "Care", "Humble"}, Flower.Type.FILLER);
-        add("Eucalyptus", "Green", new String[]{}, 1.5, new String[]{}, Flower.Type.FILLER);
+                new String[]{"Innocence", "Kindness", "Care", "Humble"}, Flower.Type.FLOWER);
+        add("Eucalyptus", "Green", new String[]{}, 1.5, new String[]{}, Flower.Type.FLOWER);
```

Comment:
Was planning to add a filler flower type to make recommend command more complex, but might scrap the idea since deadline is close